extensions/chocolatey-core.extension/extensions/Get-AvailableDriveLetter.ps1

```diff
@@ -0,0 +1,44 @@
+﻿<#
+.SYNOPSIS
+  Get a 'free' drive letter
+
+.DESCRIPTION
+  Get a not yet in-use drive letter that can be used for mounting
+
+.EXAMPLE
+  Get-AvailableDriveLetter
+
+.EXAMPLE
+  Get-AvailableDriveLetter 'X'
+  (do not return X, even if it'd be the next choice)
+
+.INPUTS
+  specific drive letter(s) that will be excluded as potential candidates
+
+.OUTPUTS
+  System.String (single drive-letter character)
+
+.LINK
+  http://stackoverflow.com/questions/12488030/getting-a-free-drive-letter/29373301#29373301
+#>
+function Get-AvailableDriveLetter {
+  param ([char[]]$ExcludedLetter)
```

Comment:
hm .. possible but I don't like it - as it isn't quite right .. 
I could specify -StartAfterLetter 'V' .. but maybe 'G' is possible - so 'G' would be returned even though I specified 'StartAfter V'